rllib/algorithms/bc/bc_learner.py

```diff
@@ -0,0 +1,25 @@
+from ray.rllib.core.learner.learner import Learner
+from ray.rllib.connectors.common.add_observations_from_episodes_to_batch import (
+    AddObservationsFromEpisodesToBatch,
+)
+from ray.rllib.connectors.learner.add_next_observations_from_episodes_to_train_batch import (  # noqa
+    AddNextObservationsFromEpisodesToTrainBatch,
+)
+from ray.rllib.utils.annotations import (
+    override,
+    OverrideToImplementCustomLogic_CallToSuperRecommended,
+)
+
+
+class BCLearner(Learner):
+    @OverrideToImplementCustomLogic_CallToSuperRecommended
+    @override(Learner)
+    def build(self) -> None:
+        super().build()
+        # Prepend a NEXT_OBS from episodes to train batch connector piece (right
```

Comment:
Cool!

Reply:
Yeah, one has to know however which connectors are needed :D

Reply:
I think it's actually fine.
* We need to document well all the off-the-shelf RLlib connectors, e.g. AddObsToBatch, Flatten, etc..
* An algo now has the chance to assemble its connector pipeline based on needs (and assumptions that certain pieces will always be there, so the algo's custom ones can be prepended/appended to these).